index.css

```diff
@@ -1,3 +1,200 @@
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_5_0.woff2") format("woff2"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_5_0.woff") format("woff"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_5_0.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Gibson";
+  font-weight: 700;
+  font-style: italic;
+  src: url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_2_0.eot");
+  src: url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_2_0.eot?#iefix") format("embedded-opentype"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_2_0.woff2") format("woff2"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_2_0.woff") format("woff"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_2_0.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Gibson";
+  font-weight: 900;
+  src: url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_3_0.eot");
+  src: url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_3_0.eot?#iefix") format("embedded-opentype"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_3_0.woff2") format("woff2"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_3_0.woff") format("woff"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_3_0.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Gibson";
+  font-weight: 900;
+  font-style: italic;
+  src: url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_1_0.eot");
+  src: url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_1_0.eot?#iefix") format("embedded-opentype"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_1_0.woff2") format("woff2"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_1_0.woff") format("woff"),
+       url("https://static.ticketfly.com/fonts/gibson/webfonts/2E3C08_1_0.ttf") format("truetype");
+}
+
+/**
+ * Base font settings
+ */
+
+:root {
+  font-size: 16px;
+
+  --base-font-color: var(--x-color-gray-8);
+}
+
+html, body {
+  color: var(--base-font-color);
+}
+
+/**
+ * Font-family settings
+ */
+
+:root {
+  --font-family-akkurat-pro:
+    Akkurat-Pro, Helvetica, Roboto, Arial, sans-serif;
+
+  --font-family-gibson: 
+    Gibson, Akkurat-Pro, Helvetica, Roboto, Arial, sans-serif;
+}
+
+html, body {
+  font-family: var(--font-family-akkurat-pro);
+}
+
+h1 {
+  font-family: var(--font-family-gibson);
+}
+
+/**
+ * Headings
+ */
+
+h1 { font-weight: 700; font-size: 2.25rem; }    /* 36px */
+h2 { font-weight: bold; font-size: 1.5rem; }    /* 24px */
+h3 { font-weight: bold; font-size: 1.125rem; }  /* 18px */
+h4 { font-weight: bold; font-size: 1rem; }      /* 16px */
+h5 { font-weight: bold; font-size: 0.875rem; }  /* 14px */
+
+/**
+ * Body Text 
```

Comment:
Missing "Meta Text" from the design spec, which should be 12px (0.75rem). Not sure which HTML element this belongs to.